src/content/docs/style-guide/writing-docs/writer-workflow/tech-writer-workflow.mdx

```diff
@@ -157,4 +157,15 @@ To create a branch on the docs-website repo:
 3. Click on **New Branch**
 4. You will be prompted to name your new branch. Descriptive names are best. It's a great way to quickly clue people in to what your work is all about. For example, if you are working on What’s New pages, you might name the branch **Whats-new-updates**.
 
-When you create a new branch, don't forget to add the Jira ticket's key (DOC-1234) to the branch name and the PR title. 
\ No newline at end of file
+When you create a new branch, don't forget to add the Jira issue ID (DOC-1234) to the branch name and the PR title.
+
+## A note about GitHub permissions [#permissions]
+
+Writers on the Docs team work in branches, but contributors outside of the Docs team use forks. This also means that non-Docs contributors can't edit content on a branch directly, which can make collaboration tricky.
```

Comment:
@paperclypse, I agree that collaboration is tricky, but this section isn't really solving for that, right? We have another pending PR for a separate doc about how to collaborate in forks (6686). Perhaps we'll need to link to that when it's ready?

Reply:
No, this is just to try and clarify how we've set up our GitHub permissions so that people understand the current set up. Happy to link to that other doc when it's ready.